<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!--Launch file for sim-->
  <include file="$(find box_bot_sim_pkg)/launch/rect_bot_world.launch"/>

  <node name="fake_angle_finder" pkg="prop_follower" type="fake_angle_finder" output="log">
    <rosparam file="$(find prop_follower)/config/params.yaml" command="load" />
  </node>

  <node name="coord_finder" pkg="prop_follower" type="coord_finder" output="log">
    <rosparam file="$(find prop_follower)/config/params.yaml" command="load" />
    <remap from="/scan" to="/rect_bot/laser/scan"/>
  </node>

  <node name="velocity_sender" pkg="prop_follower" type="velocity_sender" output="log" >
    <rosparam file="$(find prop_follower)/config/params.yaml" command="load" />
  </node>

</launch>